config/locales/en.yml

```diff
@@ -18,13 +18,20 @@ en:
     content: Content
     goals_today: Goals Today
     goals_tomorrow: Goals Tomorrow
+    description: Description
```

Comment:
I belive you should run " i18n-tasks normalize" to sort alphabetically and if your on it you can also try "i18n-tasks health " i guess